classes/order/OrderSlip.php

```diff
@@ -508,18 +508,20 @@ public function getEcoTaxTaxesBreakdown()
                 WHERE `id_order_detail` = ' . (int) $order_slip_details['id_order_detail']
             );
 
-            if (!isset($ecotax_detail[$row['rate']])) {
-                $ecotax_detail[$row['rate']] = ['ecotax_tax_incl' => 0, 'ecotax_tax_excl' => 0, 'rate' => $row['rate']];
-            }
+            if (false !== is_array($row)) {
```

Comment:
```suggestion
            if (true === is_array($row)) {
```
I think it's a little easier to read but it's just my opinion. Do you think `false !==` is better? 😄

Reply:
💯

Reply:
ying yang